libraries/botbuilder/src/skills/botFrameworkSkill.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @module botbuilder
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Registration for a BotFrameworkHttpProtocol based Skill endpoint.
+ */
+export interface BotFrameworkSkill { 
+    /**
+     * Id of the skill.
+     */
+    id?: string;
```

Comment:
I think it shouldn't be nullable (is that what the ? mean in js?)

Reply:
Made property required 👍